core/src/main/java/org/mule/runtime/core/api/event/DefaultFlowCallStack.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.api.event;
```

Comment:
can't we expose an interface instead? or at most, make this priviledged

Reply:
DW doesn't handle interfaces, only concrete classes